src/loaders/genome_attributes/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,247 @@
+"""
+PROTOTYPE
+
+This script involves processing computed genome attributes and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to update (upsert/insert) the database with the parsed
+genome attribute data.
+
+Note: If the ArangoDB collection has been previously created using a JSON file generated by the genome attributes
+      loader script, it is crucial to ensure that the arguments "--load_ver" and "--kbase_collection" are consistent
+      with the ones used in the genome attributes loader script in order to ensure that the same key is generated
+      for the corresponding Arango document."
+
+usage: parse_computed_genome_attribs.py [-h] --load_ver LOAD_VER --kbase_collection KBASE_COLLECTION
+                                        [--tools TOOLS [TOOLS ...]] [--root_dir ROOT_DIR] [-o OUTPUT]
+
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1).
+  --kbase_collection KBASE_COLLECTION
+                        KBase collection identifier name.
+
+optional arguments:
+  --tools TOOLS [TOOLS ...]
+                        Extract results from tools. (default: retrieve all available sub-directories in the [load_ver]
+                        directory)
+  --root_dir ROOT_DIR   Root directory for the collections project. (default: /global/cfs/cdirs/kbase/collections)
+  -o OUTPUT, --output OUTPUT
+                        output JSON file path (default: computed_genome_attribs.json)
+
+"""
+import argparse
+import copy
+import os
+import sys
+
+import pandas as pd
+
+from src.loaders.common import loader_common_names
+from src.loaders.gtdb.gtdb_loader_helper import convert_to_json, init_genome_atrri_doc, merge_docs
+
+# Default result file name for parsed computed genome attributes data for arango import
+COMPUTED_GENOME_ATTR_FILE = "computed_genome_attribs.json"
```

Comment:
Maybe rather than hardcoding this calculate it from the collection name and load version?

Reply:
👍

Reply:
:+1: